app/javascript/packs/red_hat_cloud_services_list.jsx

```diff
@@ -60,23 +60,42 @@ const RedHatCloudServicesList = () => {
       : null
     )
 
+    function SyncProvider(provider_id) {
+      API.post(`/api/red_hat_cloud_service_providers/${provider_id}`, {
+        action: 'sync',
+      }).then(() => dispatch({type: 'setToastVisibility', showToast: true}))
+    }
+
+    function SyncProviders(selected_providers) {
+      let providers = [];
+      selected_providers.forEach( (provider) => {
+        providers.push( { id: provider.id } )
+      });
+      API.post('/api/red_hat_cloud_service_providers', {
+        action: 'sync',
+        resources: providers,
+      }).then(() => dispatch({type: 'setToastVisibility', showToast: true}))
+    }
+
+    function SyncPlatform() {
+      API.get('/api/red_hat_cloud_service_providers?expand=resources&attributes=id&sort_by=id').then(data => {
+        if (data.resources.length > 0) {
+          SyncProviders(data.resources)
+        }
+      })
+    }
 
     useEffect(() => {
-      let labelTable = {};
-      API.options('/api/providers').then(options => {
-        labelTable = options.data.supported_providers.reduce((obj, item) => (obj[item.type] = item.title, obj) ,{});
-      }).then(
-        API.get('/api/providers?expand=resources').then(data => {
-          const rows = data.resources.map( (item) => ({
-            id: item.id,
-            name: item.name,
-            type: labelTable[item.type] || __('Unknown'),
-            action: <Button onClick={() => dispatch({type: 'setToastVisibility', showToast: true})}>Synchronize</Button>,
-            selected: false,
-          }))
-          dispatch({type: 'setRows', rows: orderBy(rows, 'name', 'asc')})
-        })
-      );
+      API.get('/api/red_hat_cloud_service_providers?expand=resources').then(data => {
```

Comment:
This will cause UX problem, in Type column will be strings like: `ManageIQ::Providers::Vmware::InfraManager`. So we need either for API to also return human readable type as a separate value or there need to be conversion table. I would prefer first solution, but the second has worked ok.

Reply:
@abellotti See the original code in that it fetches the emstype_description column.  We can just return that directly in the new API.

Reply:
pushed a UI update to fetch and display that.

![image](https://user-images.githubusercontent.com/5797328/61642219-8f2d4700-ac6e-11e9-9559-99b682ef1c72.png)